src/auth-service/utils/generate-filter.js

```diff
@@ -29,9 +29,9 @@ const filter = {
       }
       if (resetPasswordToken) {
         filter["resetPasswordToken"] = resetPasswordToken;
-        filter["resetPasswordExpires"] = {
-          $gt: Date.now(),
-        };
+        // filter["resetPasswordExpires"] = {
```

Comment:
fixing the forgot password bug from here. @BenjaminSsempala @Codebmk 

Apparently, the system is not able to do datetime comparisons when querying from the mongo DB.